pkg/controller/storagecluster/platform_detection.go

```diff
@@ -16,12 +16,16 @@ type CloudPlatformType string
 const (
 	// PlatformAWS represents the Amazon Web Services platofrm
 	PlatformAWS CloudPlatformType = "aws"
+	// PlatformGCP represents the Google cloud Platform
+	PlatformGCP CloudPlatformType = "gcp"
```

Comment:
gce?

Reply:
that is the other pr https://github.com/openshift/ocs-operator/pull/571.

Reply:
i think manual backport is required.